### PR TITLE
adding package.xml and CMakeLists.txt for rospack; package.xml has so…

### PR DESCRIPTION
**Pull request by schelian**
### What does this PR do?

makes a ros package of openface

### Where should the reviewer start?

please try to build openface_ros; the added files are more or less default files

### How should this PR be tested?

please try to build openface_ros; it will depend on this

### What are the relevant issues?

[You can link directly to issues by entering # then the
number of the issue, for example, #3 links to issue 3]

# Screenshots (if appropriate)

# Questions:

+ Do the docs need to be updated?
+ Does this PR add new (Python) dependencies?

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <!-- Author tags are optional, mutiple are allowed, one per tag -->
+  <!-- Authors do not have to be maintianers, but could be -->
+  <!-- Example: -->
+  <!-- <author email="jane.doe@example.com">Jane Doe</author> -->
+
+
+  <!-- The *_depend tags are used to specify dependencies -->
+  <!-- Dependencies can be catkin packages or system dependencies -->
+  <!-- Examples: -->
+  <!-- Use build_depend for packages you need at compile time: -->
```
Comment (athackst) [inline package.xml:25]: If you're going to catkinize this package, you should do it completely.  This file needs to include all of the dependencies of the project so we can rosdep install them.

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+#  LIBRARIES openface
+#  CATKIN_DEPENDS openface_ros
+#  DEPENDS system_lib
+)
+
+###########
+## Build ##
+###########
+
+## Specify additional locations of header files
```
Comment (athackst) [inline CMakeLists.txt:110]: I don't see any build rules here.  If you intend on catkinizing the package, it should actually build within the system.  See https://github.com/googlecartographer/cartographer for how cartographer catkinized a non-ros package.

**Review by athackst — CHANGES_REQUESTED** [review]
This looks incomplete to me.

**Comment by brandonnorthcutt-tri** [comment]
Can I get some clarification on what is required to compile and test this pull request?  I have cloned and checked out "face_rec" as described in the "view command line instructions" above. Do I now need to read through all the install instructions in the openface documentation? (It is ugly docker nonsense) Do I need to separately install torch? Is catkin supposed to produce something that can be launched with roslaunch? Is there a facial recognition unit test that I can run afterwards to see success has been achieved?

**Pull request closed without merging**